plugins/modules/monitoring/monit.py

```diff
@@ -182,15 +182,15 @@ def _parse_status(self, output, err):
             return status
 
     def is_process_present(self):
-        rc, out, err = self.module.run_command('%s summary %s' % (self.monit_bin_path, self.command_args), check_rc=True)
+        rc, out, err = self.module.run_command([self.monit_bin_path, 'summary', self.command_args], check_rc=True)
```

Comment:
command_args should be converted to a list, and this should use list concatenation.

Reply:
That's a call to `self.module.run_command()`, as opposed to one to `self.run_command()`